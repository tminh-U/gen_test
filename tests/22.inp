12
16 18 6 5 7 19 5 15 8 10 10 3 
15
9 9
8 12
5 10
12 12
11 12
8 9
11 11
9 10
12 12
3 5
5 7
7 7
8 12
11 11
1 7
