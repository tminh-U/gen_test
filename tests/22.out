2
10
17
2
3
3
2
4
2
6
5
2
10
2
20
